src/main/resources/init.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TYPE public.ticket_type AS ENUM (
+    'DAY',
+    'WEEK',
+    'MONTH',
+    'YEAR'
+);
+
+CREATE TABLE public."user_data" (
+    id uuid PRIMARY KEY NOT NULL UNIQUE,
+    name character varying NOT NULL,
+    creation_date timestamp with time zone NOT NULL
+);
+
+CREATE TABLE public."ticket_data" (
```

Comment:
the same
```sql
CREATE TABLE ticket_data (...)
```

Reply:
Removed quotes in table names and public.